sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -301,6 +302,61 @@ class SparkSqlParserSuite extends AnalysisTest {
         List.empty, List.empty, None, None, false)))
   }
 
+  test("SPARK-28227: script transform with row format delimit with aggregation") {
```

Comment:
Please add tests for window/lateral view cases.

Reply:
> Please add tests for window/lateral view cases.

Added